app/src/main/java/ltd/evilcorp/atox/ContactModel.kt

```diff
@@ -1,3 +1,3 @@
 package ltd.evilcorp.atox
 
-class ContactModel(var name: String, val toxId: String, var lastMessage: String)
+class ContactModel(var name: String, val toxId: String, var lastMessage: String, val fnum: Int)
```

Comment:
fnum -> friendNumber or friendNo (in all files)

Reply:
done